pdf-generator/build.gradle

```diff
@@ -27,4 +27,5 @@ dependencies {
   testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junit
 
   testCompile group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: springBoot
+  testCompile group: 'org.mockito', name: 'mockito-core', version: '2.+'
```

Comment:
could we use a specific version here please?

Reply:
removed as it's no longer in used